packages/console/src/components/ConfirmModal/ConfirmModal.tsx

```diff
@@ -0,0 +1,57 @@
+import { AdminConsoleKey, I18nKey } from '@logto/phrases';
+import React, { ReactNode } from 'react';
+import ReactModal from 'react-modal';
+
+import Button, { ButtonType } from '@/components/Button';
+import * as modalStyles from '@/scss/modal.module.scss';
+
+import ModalLayout from '../ModalLayout';
+import * as styles from './index.module.scss';
+
+export type ConfirmModalProps = {
+  content: ReactNode;
+  className?: string;
+  title?: AdminConsoleKey;
+  confirmButtonType?: ButtonType;
+  confirmButtonText?: I18nKey;
+  cancelButtonText?: I18nKey;
+  isOpen: boolean;
+  onCancel: () => void;
+  onConfirm: () => void;
+};
+
+const ConfirmModal = ({
+  content,
+  className = styles.content,
```

Comment:
we can merge classNames?
```
classNames(styles.content, className)
```

Reply:
I prefer to replace class names instead of merging them.
If we merge them, there may be unwanted styles inside default styles, and if we rewrite these unwanted styles, that is the same as a replcement.

Reply:
That is not aligned with other components.

Reply:
OK, updated.